aapcs64/aapcs64.rst

```diff
@@ -2965,61 +2973,65 @@ The header file ``arm_neon.h`` also defines a number of intrinsic functions that
 
 .. table:: Table 7: Short vector extended types
 
-  +-----------------+-------------------+--------------------------+-----------+
```

Comment:
Should we also add on
 Table 4, C/C++ type variants by data model
Just because I see for __fp16

Reply:
I don't think so. `__mfp8` does not vary by data model.